src/modules/site/base/utils/auth.py

```diff
@@ -15,11 +20,22 @@
                                 get_jwt_identity,
                                 get_current_user,
                                 verify_jwt_in_request,
-                                jwt_required)
+                                jwt_required,
+                                decode_token)
 
 from caendr.models.datastore import User
+from caendr.models.datastore.user_token import UserToken
 from extensions import jwt
 
+def create_one_time_token(id, destination='/'):
+  expires_delta = timedelta(seconds=JWT_LIMITED_TOKEN_EXPIRES)
+  token = create_access_token(identity=str(id), additional_claims={'destination': destination}, expires_delta=expires_delta)
+  decoded_token = decode_token(token)
+  jti = decoded_token['jti']
+  user_token = UserToken(jti)
+  user_token.set_properties(username=id, revoked=False)
```

Comment:
hum... do we need the field `revoked`? I was wondering if the mere existence of an entity in the datastore with kind=`user_expired_token` would suffice for the check. Or reframing the question, is the `revoked` field ever not `False`?